buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -351,6 +351,8 @@ class BeamModulePlugin implements Plugin<Project> {
     Integer numParallelTests = 1
     // Whether the pipeline needs --sdk_location option
     boolean needsSdkLocation = false
+    // semi_persist_dir for SDK containers
+    String semiPersistDir = "/tmp"
```

Comment:
Will this get cleaned up or accumulate junk (until a restart)?

Reply:
I assume it's safe during the lifetime of the container. We use the staging location in `/tmp` by default for publicly built containers.